src/localStorage.ts

```diff
@@ -0,0 +1,24 @@
+export const loadState = () => {
+  try {
+    const localAllTasksState: string | null = localStorage.getItem(
+      "reduxState"
+    );
+
+    if (!localAllTasksState) return undefined;
+    // when use combinedReducers()
+    // preload state must be a plain object with the same shape as the keys passed to it
+    return { allTasksReducer: JSON.parse(localAllTasksState) };
+  } catch (error) {
```

Comment:
What errors do you expect here? There is a fair amount going on in your `try` block, so this has the potential to bury some errors that you may want to know about. For your particular case, you could put a `console.error(error)` in the `catch` block.

Reply:
Added console.log(error) to the catch block, I didn't realize that I didn't have a log there. Thanks!!!